src/app/docs/alert/alert-docs.component.html

```diff
@@ -4,10 +4,269 @@
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+      >
+        <p>
+            Don’t use alerts to show messages about the success of failure of user actions. Use <a href="https://developer.blackbaud.com/skyux/components/toast">toasts</a> instead.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-usage-5.png"
+          caption="Don't use alerts to highlight the outcomes of user actions."
+          captionType="danger"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-usage-6.png"
+          caption="Do use toasts to call out the outcomes of user actions."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        imagePath="~/assets/img/guidelines/alert/alert-anatomy.png"
+      >
+        <sky-docs-anatomy-item>
+          Alert icon
+        </sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>
+          Text
+        </sky-docs-anatomy-item>
+        <sky-docs-anatomy-item
+          isOptional="true"
+        >
+          Close button
+        </sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      
+        <sky-docs-design-guideline
+        heading="Alert type"
+      >
+        <table class="sky-docs-table">
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-1.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Danger alert – for errors or to let the user know the system is in an unusable state.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-2.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Info alert – for messages that provide helpful information or context but do not require action.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-3.png"
```

Comment:
In this image, I'd add articles and a period to make this a complete sentence: "The connection to the data source is secure."

Reply:
@Blackbaud-ToddRoberts, looks like all updates to images were made except this one and one other that recommended tweaks to make the messages complete sentences. Was leaving these 2 unresolved intentional?

Reply:
nope just overlooked. should be fixed now